_posts/2019-05-01-Make-GitHub-Your-Code-Repository.md

```diff
@@ -0,0 +1,82 @@
+---
+layout: post
+title: Make GitHub Your Code Repository
+---
+
+Downloading code from [GitHub](https://github.com/)� manually is straightforward: navigate to the repository website, download the ZIP file, and extract it to your working directory.
```

Comment:
jekyll was complaining about the copyright symbol locally, but might be ok on pages ... just double check once we merge.